src/goto-instrument/havoc_utils.cpp

```diff
@@ -52,6 +59,8 @@ void havoc_utilst::append_object_havoc_code_for_expr(
   havoc.add_source_location() = location;
   havoc.add_to_operands(expr);
   dest.add(goto_programt::make_other(havoc, location));
+  dest.instructions.back().source_location_nonconst().set_comment(
+    "Loop havocing instrumented by appling loop contracts");
```

Comment:
If you really need to use a string comment to communicate this information, then please at least put this string in a macro. (And perhaps fix the typo: s/appling/applying/)